Test:  scripts/funcTests/runFuncTests.sh fails on macOS

[This commit](https://github.com/NuGet/NuGet.Client/commit/6bbf84b17503b7c3de15edb5b639457e92cc402e) introduced an error in scripts/funcTests/runFuncTests.sh.  Example:
```
Cleaning TMP directory
rm: /tmp/dotnet.*: No such file or directory
rm: -rf: No such file or directory
=================
/Users/buildagent/agent/_work/32/s/cli/dotnet msbuild build/build.proj /t:Restore /p:VisualStudioVersion=15.0 /p:Configuration=Release /p:BuildNumber=1 /p:ReleaseLabel=beta
/Users/buildagent/agent/_work/32/s/scripts/funcTests/runFuncTests.sh: line 80: /Users/buildagent/agent/_work/32/s/cli/dotnet: No such file or directory
Restore failed!!
##[error]/bin/bash failed with return code: 1
##[error]Bash failed with error: /bin/bash failed with return code: 1
```
The reason is that the syntax for the `rm` command puts options before file/directory paths.  Linux is lenient and allows options after paths but macOS is not.

CC @donnie-msft

--- a/scripts/funcTests/runFuncTests.sh
+++ b/scripts/funcTests/runFuncTests.sh
@@ -54,9 +54,8 @@ cli/dotnet-install.sh -i cli -c $DOTNET_BRANCH
 # Display current version
 $DOTNET --version
 
-# Cleanup TMP directory
-echo "Cleaning TMP directory"
-rm "/tmp/"dotnet.* -rf
+echo "Deleting .NET Core temporary files"
+rm -rf "/tmp/"dotnet.*
 
 echo "================="
 

